client/app/components/form/widgets/FileWidget.tsx

```diff
@@ -109,11 +163,20 @@ const FileWidget = ({
   const handleChange = useCallback(
     (event: ChangeEvent<HTMLInputElement>) => {
       event.preventDefault();
-      if (!event.target.files) {
+      const maxSize = 1024 * 1024; // 1 MB in bytes
```

Comment:
Next uses 1MB as the default, so I've gone with that until I hear otherwise from Nan. We need to change it here and in the next config: By default body size limit is 1MB, we can change it here: https://nextjs.org/docs/app/api-reference/next-config-js/serverActions#bodysizelimit

Reply:
20MB